libs/coin-modules/coin-tron/src/network/index.ts

```diff
@@ -647,7 +651,7 @@ export async function getTronResources(
     .dividedBy(1000000)
     .integerValue(BigNumber.ROUND_FLOOR)
     .toNumber();
-  const votes = get(acc, "votes", []).map(v => ({
+  const votes = get(acc, "votes", []).map((v: any) => ({
```

Comment:
same I don't think it's relevant to call this function just to do `(acc.votes || []).map`